assets/src/amp-validation/copy-to-clipboard-buttons.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
Alternatively you could use [clipboard.js](https://clipboardjs.com/) (which comes bundled with WordPress), but this approach is also fine :+1:.

Reply:
Good suggestion. I didn't think of this. I've refactored out the wheel-reinvention in 61e571a47